src/main/java/racinggame/NumberVo.java

```diff
@@ -0,0 +1,53 @@
+package racinggame;
+
+import java.util.Random;
+
+/**
+ * 1. 0~9 사이의 random 값을 구하는 기능
+ * 1. 값이 정수 인지 체크 및 반환
+ * 1. 값이 1 이상인치 체크
+ * 1. 값이 4 이상인치 체크
+ */
+public class NumberVo {
+
+    private NumberVo() {
+    }
+
+    /**
+     * 0~9 사이의 random 값을 구하는 기능
+     */
+    protected static int createRandomNumber() {
+        return new Random().nextInt(10);
+    }
+
+    /**
+     * 값이 4 이상인치 체크
+     */
+    protected static boolean isGreaterThan4(int num) {
+        if (num >= 4) {
+            return true;
+        }
+        return false;
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return num >= 4;
```

Reply:
좋은 지적 감사합니다! 😃